software/chartdb.yml

```
@@ -0,0 +1,12 @@
+name: ChartDB
+website_url: https://chartdb.io/
+description: Database diagrams editor that allows you to visualize and design your DB with a single query. 
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+  - Docker
+tags:
+  - Miscellaneous
```

```suggestion
  - Database Management
  - Miscellaneous
```
I could be wrong, but doesn't this count under database management? It fits the description in a way. otherwise, feel free to ignore this. Otherwise, LGTM

---

Good catch!